app/views/estimatedIncomeTax/NoCurrentIncomeView.scala.html

```diff
@@ -43,18 +43,25 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = Messages("tai.estimatedIncome.title"),
         preHeadingText = taxYearRangeString,
-        preHeadingAccessibleAnnouncement = Messages("tai.estimatedIncome.accessiblePreHeading")
+        preHeadingAccessibleAnnouncement = Messages("tai.estimatedIncome.accessiblePreHeading"),
+        showGdsVersion = true
     )
 }
 
 @mainTemplate(
     title = Messages("tai.estimatedIncome.title"),
-    pageHeader = Some(header),
-    articleClasses = Some("full-width"),
-    authedUser = Some(user)
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    authedUser = Some(user),
+    backLinkId = "backLink",
+    backLinkContent = Some(Messages("tai.back-link.upper")),
```

Comment:
the old JS backLink is being used, could we not set the URL to http://localhost:9230/check-income-tax/income-summary as it looks like this NoCurrentIncomeView.scala.html is only used in one place

Reply:
I thought because it doesn't have any forms or anything it could just use JS as no errors should appear

Reply:
btw... there is a ticket in QA that (PR  https://github.com/hmrc/tai-frontend/pull/992/files)   that changes the default backlink to..   backLinkId  in the new main template